common/src/models/service.rs

```diff
@@ -200,3 +210,52 @@ fn get_secrets_table(secrets: &[&resource::Response]) -> String {
 "#,
     )
 }
+
+fn get_static_folder_table(static_folders: &[&resource::Response]) -> String {
+    let mut table = Table::new();
+
+    table
+        .load_preset(UTF8_FULL)
+        .apply_modifier(UTF8_ROUND_CORNERS)
+        .set_content_arrangement(ContentArrangement::DynamicFullWidth)
+        .set_header(vec![
+            Cell::new("Static Folders").set_alignment(CellAlignment::Center)
+        ]);
+
+    for folder in static_folders {
+        let path = serde_json::from_value::<PathBuf>(folder.data.clone())
+            .unwrap()
+            .display()
+            .to_string();
+
+        table.add_row(vec![path]);
+    }
+
+    format!(
+        r#"These static folders can be accessed by the service
+{table}
+"#,
+    )
+}
+
+fn get_persist_table(persist_instances: &[&resource::Response]) -> String {
+    let mut table = Table::new();
+
+    table
+        .load_preset(UTF8_FULL)
+        .apply_modifier(UTF8_ROUND_CORNERS)
+        .set_content_arrangement(ContentArrangement::DynamicFullWidth)
+        .set_header(vec![
+            Cell::new("Instances").set_alignment(CellAlignment::Center)
+        ]);
+
+    for (i, _) in persist_instances.iter().enumerate() {
+        table.add_row(vec![format!("Persist Instance {i}")]);
+    }
+
+    format!(
+        r#"These instances are linked to this service
+{table}
+"#,
+    )
+}
```

Comment:
This one I'm not so sure about how we should handle, since the output just contains the project name. :thinking: